### PR TITLE
feat: replaced cube primitives with 3D crop model

### PR DESCRIPTION
**Pull request by wifimug**

### CODE REVIEW

**Review by SeanChao — APPROVED** [review]
LGTM! I would recommend removing the commented code.

File: src/controllers/camera_controller.py [hunk 1]
```diff
@@ -5,18 +5,21 @@ class CameraController:
...
     """
-    def setup_camera(self, cam="camera_one", camera_location=(10,0,0),
+    def setup_camera(self, cam="camera_one", camera_location=(10,5,0),
                      camera_rotation=(1.57057,0.00174533,1.57057), collection_name="Collection"):
         ## temporarily removing the existing cube object in the blend file
-        collection1 = bpy.data.collections.get("Collection")
-        cube = collection1.objects.get("Cube")
-        collection1.objects.unlink(cube)
+        # bpy.ops.wm.open_mainfile(filepath="src/blender_assets/CropAssets.blend")
+        # bpy.ops.object.camera_add(enter_editmode=False, align="VIEW",
```
Comment (DanielSwarup) [inline src/controllers/camera_controller.py:12]: Would recommend removing unused code.

**Review by DanielSwarup — APPROVED** [review]

**Comment by DanielSwarup** [comment]
Would recommend writing unit tests as well

**Comment by wifimug** [comment]
Unit tests incoming!

**Pull request merged**